ovs/dal/dataobject.py

```diff
@@ -736,7 +739,7 @@ def delete(self, abandon=None, _hook=None):
                 self._persistent.apply_transaction(transaction)
                 successful = True
             except KeyNotFoundException as ex:
-                if ex.message != self._key:
+                if self._key not in ex.message:
```

Comment:
`KeyNotFoundException: 'ovs_data_vdisk_55e15197-183f-43dd-b25c-ef2425d6241e'

In [4]: %debug
> /opt/OpenvStorage/ovs/extensions/storage/persistent/pyrakoonstore.py(224)apply_transaction()
    223         except ArakoonNotFound as field:
--> 224             raise KeyNotFoundException(field)
    225 

ipdb> up
> /opt/OpenvStorage/ovs/dal/dataobject.py(740)delete()
    739                 if ex.message != self._key:
--> 740                     raise
    741                 successful = True

ipdb> ex.message
ArakoonNotFound('ovs_data_vdisk_55e15197-183f-43dd-b25c-ef2425d6241e',)
ipdb> self._key
'ovs_data_vdisk_55e15197-183f-43dd-b25c-ef2425d6241e'
ipdb> 
`